setup.py.in

```diff
@@ -55,7 +52,7 @@ print('src:',srcfiles)
 print('macros:',macros)
 
 setup (name = "CMOR",
-       version='3.6.1',
+       version='3.7.0',
        author='Denis Nadeau, AIMS',
```

Comment:
@mauzey1 you may as well update yourself here too - and anywhere else deprecated information resides